### PR TITLE
Incorporate "Use StatsD" information

### PR DESCRIPTION
**Pull request by beccatortell**

### CODE REVIEW

File: collectd-statsd/README.md [hunk 1]
```diff
@@ -51,47 +53,79 @@ This plugin requires:
...
-Configuration for this plugin is kept in the main [collectd.conf](https://github.com/signalfx/integrations/blob/master/collectd/collectd.conf) file.
+SignalFx's example configuration file for this plugin can be used as-is, without modification. To read more about available configuration options, please see the [collectd wiki](https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_statsd).
 
-From the [collectd wiki](https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_statsd):
+### USAGE
 
-> The statsd plugin listens to a UDP socket, reads "events" in the statsd protocol and dispatches rates or other aggregates of these numbers periodically.
+#### Deployment options
 
-> The plugin implements the _Counter_, _Timer_, _Gauge_ and _Set_ types which are dispatched as the collectd types `derive`, `latency`, `gauge` and `objects` respectively.
```
Comment (mdubbyap) [inline collectd-statsd/README.md]: This seems like useful information, why remove it?

File: collectd-statsd/README.md [hunk 2]
```diff
@@ -51,47 +53,79 @@ This plugin requires:
...
-|TimerLower |false/true| Calculate and dispatch various values out of Timer metrics received during an interval. If set to False, the default, these values aren't calculated / dispatched.|
-|TimerUpper |false/true| Calculate and dispatch various values out of Timer metrics received during an interval. If set to False, the default, these values aren't calculated / dispatched.|
-|TimerSum |false/true| Calculate and dispatch various values out of Timer metrics received during an interval. If set to False, the default, these values aren't calculated / dispatched. |
-|TimerCount |false/true|Calculate and dispatch various values out of Timer metrics received during an interval. If set to False, the default, these values aren't calculated / dispatched.|
```
Comment (mdubbyap) [inline collectd-statsd/README.md]: are we removing this because it's already in the collectd wiki?

Comment (beccatortell) [inline collectd-statsd/README.md]: Yes.

File: collectd-statsd/README.md (hunk 1 above)
Comment (beccatortell) [inline collectd-statsd/README.md]: Same as below. But at second glance this does seem useful to the casual user.

**Comment by mdubbyap** [comment]
lgtm

**Pull request merged**